src/keeper/contracts/DIDRegistry.ts

```diff
@@ -149,7 +149,7 @@ export default class DIDRegistry extends ContractBase {
         ownerAddress: string,
         url: string,
         immutableUrl = '',
-        activityId = Assets.DEFAULT_REGISTRATION_ACTIVITY_ID,
+        activityId = AssetsApi.DEFAULT_REGISTRATION_ACTIVITY_ID,
```

Comment:
We should probably have a provenance module for the provenance methods and this constants

Reply:
:+1: